src/index.js

```diff
@@ -93,8 +93,8 @@ export default (userOptions: UserOptionsType = {}): Object => {
         return false;
       }
 
-      if (_.has(compiler, 'options.output.path') && compiler.options.output.path !== '/') {
-        outputPath = compiler.options.output.path;
+      if (options.outputPath) {
+        outputPath = options.outputPath;
```

Comment:
I believe this will cause the plugin to fail if no `outputPath` option is specified (see the check below this). For backwards compatibility it may was to fall back to the original `output.path` of the compiler.